dojo/tools/acunetix360/parser.py

```diff
@@ -17,14 +17,24 @@ def get_description_for_scan_types(self, scan_type):
         return "Acunetix360 JSON format."
 
     def get_findings(self, filename, test):
-        data = json.load(filename)
+        tree = filename.read()
+        try:
+            data = json.loads(str(tree, 'utf-8-sig'))
+        except:
+            data = json.loads(tree)
```

Comment:
json.load() method support uTF8 / UTF16 wi or without bom.
This change break this support.

Reply:
rollback to previous version